### PR TITLE
feat(client): introduce core-js

### PR DESCRIPTION
**Pull request by hiroppy**
- [ ] This is a **bugfix**
- [x] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

modify only

### Motivation / Use-Case

https://github.com/webpack/webpack-dev-server/pull/1948#discussion_r289607053

### Breaking Changes

no

### CODE REVIEW

File: client-src/default/polyfill.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+require('core-js/features/string/includes');
+require('core-js/features/array/includes');
```
Comment (evilebottnawi) [inline client-src/default/polyfill.js]: We pollution global namespace :disappointed: We should use name import like in docs https://github.com/zloirock/core-js

Comment (hiroppy) [reply client-src/default/polyfill.js]: sorry, why?

Comment (evilebottnawi) [reply client-src/default/polyfill.js]: Maybe we need setup support `import`/`export` in our babel config for client too

Comment (evilebottnawi) [reply client-src/default/polyfill.js]: Using `require('core-js/features/array/includes');` modify `Array.prototype`, so if developer already  have modified `Array.prototype` we break can break their app. Using `import arrayIncludes from 'core-js/features/array/includes'` don't modify `Array.prototype` so it is safe to use

Comment (hiroppy) [reply client-src/default/polyfill.js]: Makes sense. ok, I'll change

**Comment by hiroppy** [comment]
@evilebottnawi PTAL

**Comment by hiroppy** [comment]
I'll create other new pr.

**Pull request closed without merging**